utils/validator.js

```diff
@@ -48,3 +49,26 @@ export const isAdmin = () => {
         next();
     };
 };
+
+export const isNotParents = () => {
```

Comment:
I am not sure we need this in real app because there will not be edit button .. but may be useful in testing api